### PR TITLE
Move to our new licensing regime.

### PR DESCRIPTION
**Pull request by ltratt**
All copyright in this project is held by King's, so moving it is trivial.

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The following contributors wish to explicitly make it known that the copyright
+of their contributions is retained by an organisation:
+
+    Edd Barrett <vext01@gmail.com>: copyright retained by
+      King's College London
+    Jacob Hughes <jh@jakehughes.uk>: copyright retained by
+      King's College London
+    Laurence Tratt <laurie@tratt.net>: copyright retained by
+      King's College London
```
Comment (ptersilie) [inline COPYRIGHT]: You forgot to add me to this copyright notice, which is ironic given that this repo was written entirely by myself. ;-)

Comment (ltratt) [reply COPYRIGHT]: Oops! Fixing.

Comment (ptersilie) [reply COPYRIGHT]: Also, unless you want to keep them there in case they make changes in the future, Edd and Jake don't need to be here.

Comment (ltratt) [reply COPYRIGHT]: Fixed in https://github.com/softdevteam/sparsevec/pull/11/commits/34faea2f8e53cb18b316119d656c00491bac5156

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Review by ptersilie — APPROVED** [review]

**Pull request merged**